hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/HoodieHiveClient.java

```diff
@@ -69,6 +70,8 @@
   private static final Logger LOG = LogManager.getLogger(HoodieHiveClient.class);
   private final PartitionValueExtractor partitionValueExtractor;
   private IMetaStoreClient client;
+  private SessionState sessionState;
+  private org.apache.hadoop.hive.ql.Driver hiveDriver;
```

Comment:
Not really an issue from this PR. but do we need to really specify the full package name? `org.apache.hadoop.hive.ql.Driver`?

Reply:
thank you, fixed